projects/imagenet/experiments.py

```diff
@@ -139,42 +135,53 @@
 
 DEFAULT100 = copy.deepcopy(DEFAULT)
 DEFAULT100.update(
-    epochs=100,
+    epochs=70,
     num_classes=100,
-    init_batch_norm=False,
-
-    # Create default sparse network
     model_args=dict(config=dict(num_classes=100, defaults_sparse=False)),
 )
 
-# Use normal schedule
+DEFAULT1000 = copy.deepcopy(DEFAULT100)
+DEFAULT1000.update(
+    epochs=70,
+    num_classes=1000,
+    model_args=dict(config=dict(num_classes=1000, defaults_sparse=False)),
+)
+
+# Use normal schedule. This currently gets to about 80.1% in 70 epochs.
 SPARSE100 = copy.deepcopy(DEFAULT100)
 SPARSE100.update(
-    # Create default sparse network
     model_args=dict(config=dict(num_classes=100, defaults_sparse=True)),
 )
 
-# Use plain learning schedule, i.e. with no momentum
-SPARSE100_PLAIN_LEARNING = copy.deepcopy(SPARSE100)
-SPARSE100_PLAIN_LEARNING.update(
+SPARSE1000 = copy.deepcopy(DEFAULT100)
+SPARSE1000.update(
+    epochs=70,
+    num_classes=1000,
```

Comment:
We dont need to set num_classes to 1000. It is the default value

Reply:
All these configs and parameters inherit from the 100 category config (because some of the hyper parameter tuning is done there).